torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2045,6 +2045,30 @@ def fill_output_arg(
 
         return new_tensor_args, new_int_args
 
+    @staticmethod
+    def _compatible_with_stableivalue(op: torch._ops.OpOverload) -> bool:
+        """Returns true if op_overload._schema is compatible with StableIValue, as
+        defined in the common C-shim."""
```

Comment:
@benjaminglass1 for now could you update this description to say that it is based on what is supported in to_ivalue (like you explained in the comment) and that this function is aware that to_ivalue is an internal implementation detail? In the future it would be good to align this function with the documentation.

Reply:
@janeyx99 Sorry it took me so long to get back to this; done!